### ISSUE TITLE
Wishlist: Warn on adding Quest Items + Allow Remove/Destroy Quest Items temporary flag

### ISSUE THREAD

**Issue by NeatWolf** [SolastaMods/SolastaUnfinishedBusiness#2915]
Hi, I accidentally added some quest items, mistakenly thinking they were common items.

- One time it happened on respec: my wizard got a duplicate spellbook. This is... tolerable as at some point I may need another. More like a nuisance.
- Another time via the SUB interface, I admit I was a bit careless but nothing led me to think they were quest items before picking them up.

Would it be possible to either, or both:

- show a warning dialogue when adding quest items to the inventory (or, hide them by default, maybe have a flag to enable adding/showing them) with a simple Yes/No to prevent adding it accidentally (not everyone's already finished the game)
- allow a debug/tool/experimental/beta flag that - temporarily allows quest items to be dropped from the inventory

Just an idea - I can't recover my save so that would really be helpful to me/others+appreciated!
I just hope it's not going to break my savegame (and whole campaign) whenever those items are supposed to trigger or be checked by the quest logic.

Thanks for your work!

**Comment by MartinBrandstrup** [comment]
This would be a nice addition since I actually had a situation where I played a custom campaign and those pesky minotaurs decided to drop unused quest item Minotaur Head with a weight of 5. They were still flagged as quest items, so if I accidentally picked them up they would be forever stuck in my inventory xD

Of course this is a mistake on the campaign maker's side and not the mod, but it would still be a nice qol feature.

Also this should probably be converted to a discussion @NeatWolf

**Comment by NeatWolf** [comment]
> This would be a nice addition since I actually had a situation where I played a custom campaign and those pesky minotaurs decided to drop unused quest item Minotaur Head with a weight of 5. They were still flagged as quest items, so if I accidentally picked them up they would be forever stuck in my inventory xD
> 
> Of course this is a mistake on the campaign maker's side and not the mod, but it would still be a nice qol feature.
> 
> Also this should probably be converted to a discussion @NeatWolf

I agree, I only noticed the Discussion area one moment too late - but I felt that to be quite relevant as it's a very easy way to compromise days of play and while I understand it's for many just something QoL, I'm a bit on the edge as I can't solve it and I fear when in the campaign the issues may pop up.

The post-respec spellbook (quest item) duplication bug is definitely something to tackle as well, so I was hoping in getting both of them sorted in a hacky way. Dropping them on the ground isn't classy but I can't think of an easier way to dispose of them if not enabling a momentary flag in the SUB ui that hooks either into the dropping obj function, or in the vendor/selling function.

I mean, if there was something like a savegame/inventory editor I'd use that but that's a lot to ask, I may have finished the game by then. Or I may get stuck on some quest forever due to the extra objs ^^

**Comment by NeatWolf** [comment]
Idea:

Have a duplicate "Identify" spell scroll that overrides just a few functions - instead of requiring to pick a magic item, it targets any item with the Quest tag and deletes it from the inventory.
